src/ansys/units/quantity.py

```diff
@@ -336,23 +419,23 @@ def __float__(self):
             dims(dimensions={base_dims.SOLID_ANGLE: 1.0}),
         ]:
             return get_si_value(self)
+        return self.value
```

Comment:
@Gobot1234 I missed this part. We should not return `self.value` here. I have made earlier comments with respect to `def __float__(self: Quantity[float]) -> float:` change only. I had not considered `return self.value` change at that time. Please remove `return self.value` and keep `def __float__(self: Quantity[float]) -> float:`.

Reply:
Ah right yeah I'll just add that in the error message